incubator/hnc/pkg/controllers/hierarchy_controller.go

```diff
@@ -41,7 +41,7 @@ import (
 // as well as ensuring all objects in the forest are propagated correctly when the hierarchy
 // changes. It can also set the status of the Hierarchy CRs, as well as (in rare cases) override
 // part of its spec (i.e., if a parent namespace no longer exists).
-type HierarchyReconciler struct {
+type HierarchyConfigReconciler struct {
```

Comment:
As mentioned above, please revert this change - it's only the API names that are changing, this is just an internal data structure.

Reply:
Another thing I've learned is the power of the namespace. When I started this I was under the assumption that if they were named close to the same they were part of the same process. This shows it not to be so, and teaches me more about how unit testing works. Ie mixing types and seeing what blows up.